vars/virtualenv.groovy

```diff
@@ -0,0 +1,14 @@
+def create(String python, String path = "venv", boolean installRequirements = true) {
+  sh("virtualenv -p ${python} ${path}")
```

Comment:
Let's use the long option `--python=${python}`.

Reply:
I don't like `path` being a default parameter, it's almost equivalent to hard-coding the value. Also I don't like `path` as a name, it's too generic. `virtualenv`'s help uses `dest_dir`.